lib/indexable_content_checker.rb

```diff
@@ -0,0 +1,71 @@
+require 'gds_api/content_store'
```

Comment:
Done. Also renamed the `y` variable to `yielder`, which is the terminology that `Enumerator` uses.